### DISCUSSION TITLE
Android Geolocation with GooglePlayServices

### DISCUSSION THREAD

**Discussion by eamonn-alphin** [dotnet/maui#4315] [Q&A]
Sorry if this isn't allowed, I wasn't sure where else to ask.

I'm converting an existing app from Xamarin to Maui and I can't seem to figure out how to implement continuous geolocating. 
In Xamarin, I used the Xamarin.GooglePlayServices.Location and related packages, but I'm wary to do so in MAUI in case support for them suddenly drops. I also don't want to put the Maui.Essentials geolocator methods in an interval timer. 

Is a Maui.GooglePlayServices set of nuget packages on the way, will the Xamarin.GooglePlayServices packages continue to be supported, or is there something I missed?

Thanks!

**Comment by BobbyCannon** [comment]
I'm surprised there's no answer on this?
